legwork/source.py

```diff
@@ -211,7 +214,7 @@ def max_strain_harmonic(e):   # pragma: no cover
 
         self.max_strain_harmonic = max_strain_harmonic
 
-    def find_eccentric_transition(self):
+    def fe_maskentric_transition(self):
```

Comment:
I think there may be a find and replace situation going on here?

Reply:
Oof nice catch, I'll fix these now!